item_names.lua

```diff
@@ -3,6 +3,8 @@
 local item_names = {} -- [player_name] = { hud, dtime, itemname }
 local dlimit = 3  -- HUD element will be hidden after this many seconds
 local hudbars_mod = minetest.get_modpath("hudbars")
+local only_names = minetest.settings:get_bool("unified_inventory_only_names")
```

Comment:
Unlike indicated in settingtypes.txt, this setting does not default to `true`. If you would like to use that default value, consider one of the following:

```suggestion
local only_names = minetest.settings:get_bool("unified_inventory_only_names") ~= false
```
or
```suggestion
local only_names = minetest.settings:get_bool("unified_inventory_only_names", true)
```